registration/include/pcl/registration/ppf_registration.h

```diff
@@ -64,7 +66,7 @@ namespace pcl
           this->second.second.second = d;
         }
       };
-      typedef boost::unordered_multimap<HashKeyStruct, std::pair<size_t, size_t> > FeatureHashMapType;
+      typedef std::unordered_multimap<HashKeyStruct, std::pair<size_t, size_t>, boost::hash<HashKeyStruct>> FeatureHashMapType;
```

Comment:
The std library does not expose hashing specializations for containers. Since we're still using boost, I leveraged it.

Reply:
Unfortunately, does not seem to work and I don't have any other proposals. Maybe leave `boost::unordered_multimap` for now?

Reply:
I haven't got back to this once I created the PR. I have one alternative in mind but I still need to look with special care to the error message.

Note that in cppreference even mentions boost::hash for hashing the std::pair container.

> Note: additional specializations for std::pair and the standard container types, as well as utility functions to compose hashes are available in boost.hash
Source: https://en.cppreference.com/w/cpp/utility/hash